app/gui/view/src/notification/logged.rs

```diff
@@ -0,0 +1,180 @@
+//! The API is often inconvenient as it returns [`Result`]s with [`JsValue`] errors. This is
+//! inconvenient for our typical use-cases. We often use notifications to display errors and
+//! we don't have a way to handle errors from the notifications themselves.
+//!
+//! This module provides a notification API wrapper that:
+//! * Logs most of the actions.
+//! * Handles errors by logging and dropping them.
+//!
+//! In general, where the original API returns [`Result`]s, this wrapper returns [`Option`]s.
```

Comment:
The second paragraph should be first - we should start with summary about the module, and add explanations later. Also, now I'm greeted with information "The API is often inconvenient" and I ask myself "What API?".

The "inconvenient" word is a bit repetitive in the first paragraph.

Reply:
fixed